src/main/java/spoon/support/compiler/jdt/JDTTreeBuilderHelper.java

```diff
@@ -471,7 +471,9 @@ public boolean onAccess(char[][] tokens, int index) {
 					qualifiedNameReference.actualReceiverType.getPackage(),
 					qualifiedNameReference, fieldReference.getSimpleName(), typeAccess.getAccessedType());
 		} else {
-			typeAccess.setImplicit(qualifiedNameReference.isImplicitThis());
+			if (qualifiedNameReference.isImplicitThis() || qualifiedNameReference.tokens.length == 2) {
```

Comment:
What does the 2 for the tokens.length mean?

Reply:
I added a comment.